clang/test/AST/HLSL/StructuredBuffer-AST.hlsl

```diff
@@ -17,7 +17,7 @@
 // EMPTY-NEXT: FinalAttr 0x{{[0-9A-Fa-f]+}} <<invalid sloc>> Implicit final
 
 // There should be no more occurrances of StructuredBuffer
-// EMPTY-NOT: StructuredBuffer
+// EMPTY-NOT: {{/s}}StructuredBuffer

```

Comment:
This ensures that StructuredBuffer shows up at the start of the line somewhere?

Reply:
This ensures that we match StructuredBuffer with a space in front of it. Avoids match with RWStructuredBuffer.

Reply:
How do you know that "/s" is space?  I didn't know that, and I can't find that in POSIX extended regex docs.  I would think you need to use character class like `[:space:]` instead.

Reply:
Oh yeah, I think you could also use `{{[^W]}}` to exclude matches of `WStructuredBuffer` instead of specifically looking for space.  What if you have `'StructuredBuffer ...'`, with `'` or something other than space right before it?

Reply:
"/s" is a space in the flavor of regex I learned years ago :) I also found it defined here:
https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference

Reply:
I've update it to use `{^\W}}`.